src/main/java/uk/nhs/digital/nhsconnect/nhais/mapper/PersonAddressMapper.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.digital.nhsconnect.nhais.mapper;
+
+import org.hl7.fhir.r4.model.Address;
+import org.hl7.fhir.r4.model.Parameters;
+import org.hl7.fhir.r4.model.Patient;
+import org.hl7.fhir.r4.model.StringType;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.PersonAddress;
+
+import java.util.List;
+
+public class PersonAddressMapper implements FromFhirToEdifactMapper<PersonAddress> {
+    private final static Address.AddressUse ADDRESS_USE_HOME = Address.AddressUse.HOME;
+
+    public PersonAddress map(Parameters parameters) {
+        Address address = getAddress(parameters);
+
+        return PersonAddress.builder()
+                .addressText(address.getText())
+                .addressLine1(getAddressLineOrNull(address.getLine(), 0))
+                .addressLine2(getAddressLineOrNull(address.getLine(), 1))
+                .build();
+    }
+
+    private Address getAddress(Parameters parameters) {
+        Patient patient = getPatient(parameters);
+
+        return patient.getAddress().stream()
+                .filter(address -> address.getUse().equals(ADDRESS_USE_HOME))
```

Comment:
We need to share code style as you have something wrong set up. There are 8 spaces instead of 4